GitExtUtils/GitUI/UIExtensions.cs

```diff
@@ -43,5 +45,27 @@ public static bool IsFixedWidth(this Font ft, Graphics g)
 
             return fixedWidth;
         }
+
+        public static string? FormatBodyAndNotes(string? body, string? notes)
+        {
+            if (string.IsNullOrWhiteSpace(notes))
+            {
+                return body;
+            }
+
+            const string notesPrefix = "Notes:";
+            const string indent = "    ";
+            StringBuilder? sb = new StringBuilder((body?.Length ?? 0) + 2 + notesPrefix.Length + 2 + indent.Length + notes.Length + 1)
+                .AppendLine(body)
+                .AppendLine(notesPrefix);
+
+            foreach (string? line in notes.Split('\n'))
+            {
+                sb.Append(indent).Append(line).Append('\n');
+            }
+
+            sb.Length--;
```

Comment:
nit:
```suggestion
            --sb.Length;
```
The name "C++" was an unlucky, misleading choice.

Reply:
In the whole GE codebase 62% of such decrements and 58% of increments and are "post". So, trying to blend in, I shall be using post-decrement. Compiler and JIT make no difference, as the value is not used anywhere.

Reply:
GE has a long history. Please do not argue with examples of former code style.

Reply:
ok